### PR TITLE
[config] Add regression message for Conan v2 builds

### PR DESCRIPTION
**Pull request by uilianries**
When a package which is listed in https://github.com/conan-io/conan-center-index/blob/master/.c3i/conan_v2_ready_references.yml fails to build on a new PR, it will be considered a regression error, which means, for that specific package, Conan v2 will be mandatory.

However, the current information on the PR, informs that Conan v2 is only informative and is not mandatory, creating a misinformation.

We are preparing a new feature which will update the message according the situation, so users will not be confused by the build result message.

---

- [ ] I've read the [contributing guidelines](https://github.com/conan-io/conan-center-index/blob/master/CONTRIBUTING.md).
- [ ] I've used a [recent](https://github.com/conan-io/conan/releases/latest) Conan client version close to the [currently deployed](https://github.com/conan-io/conan-center-index/blob/master/.c3i/config_v1.yml#L6).
- [ ] I've tried at least one configuration locally with the [conan-center hook](https://github.com/conan-io/hooks.git) activated.

### CODE REVIEW

**Review by jwillikers — DISMISSED** [review]

File: .c3i/config_v2.yml [hunk 1]
```diff
@@ -37,6 +37,7 @@ tasks:
     user_feedback:
       title: "Conan v2 pipeline (informative, not required for merge)"
```
Comment (prince-chrismc) [inline .c3i/config_v2.yml:38]:
```suggestion
      title: "Conan v2 pipeline"
```

File: .c3i/config_v2.yml [hunk 2]
```diff
@@ -37,6 +37,7 @@ tasks:
...
       description: "> **Note**: Conan v2 builds are informative and they are not required for the PR to be merged."
```
Comment (prince-chrismc) [inline .c3i/config_v2.yml:39]:
```suggestion
      description: "> **Note**: Conan v2 builds may be required once they are on the [v2 required reference]() list"
```

**Review by prince-chrismc — COMMENTED** [review]
We should update the comment from the bot too

File: .c3i/config_v2.yml (hunk 1 above)
Comment (uilianries) [reply .c3i/config_v2.yml:38]: It's optional by default, only for those which are listed in the conan_v2_ready_references.yml they mandatory. It's better keeping it explicit, removing the informative section may cause confusion.

Comment (uilianries) [reply .c3i/config_v2.yml:38]: Sorry, yes, the `title` field needs to be updated.

**Review by prince-chrismc — APPROVED** [review]

File: .c3i/config_v2.yml (hunk 1 above)
Comment (prince-chrismc) [reply .c3i/config_v2.yml]: Yes I put it in the description with a link :)

I think it's the right time to be more pushy about supporting v2

Comment (uilianries) [reply .c3i/config_v2.yml]: > I think it's the right time to be more pushy about supporting v2

Agreed. :)

File: .c3i/config_v2.yml [hunk 3]
```diff
@@ -35,8 +35,9 @@ tasks:
     write_comments: false
     update_labels: false
     user_feedback:
-      title: "Conan v2 pipeline (informative, not required for merge)"
-      description: "> **Note**: Conan v2 builds are informative and they are not required for the PR to be merged."
+      title: "Conan v2 pipeline"
+      description: "> **Note**: Conan v2 builds may be required once they are on the [v2 ready](https://github.com/conan-io/conan-center-index/blob/master/.c3i/conan_v2_ready_references.yml) list"
```
Comment (SSE4) [inline .c3i/config_v2.yml:39]: why `may be`? not always?

**Review by SSE4 — APPROVED** [review]

File: .c3i/config_v2.yml (hunk 3 above)
Comment (prince-chrismc) [reply .c3i/config_v2.yml:39]: There's a missing comma but I should be fine, the intention is clear about the condition

**Pull request merged**